### ISSUE TITLE
initOAuth is not called

### ISSUE THREAD

**Issue by Naid405** [scttcper/koa2-swagger-ui#15]
Subj, it's not being called during UI init

**Comment by scttcper** [comment]
I didn't add it back with the switch to swagger v2. Not sure how it works.

**Comment by scttcper** [comment]
I guess docs are https://github.com/swagger-api/swagger-ui#swaggeruibundle might look into it.

**Comment by Naid405** [comment]
The following worked for me:
index.hbs
```
// Build a system
  const ui = SwaggerUIBundle(options)
  if(ui.hasOwnProperty("initOAuth"))
    ui.initOAuth({{{json oauthOptions}}})
  window.ui = ui
```

**Comment by scttcper** [comment]
Thanks published under 2.4.4
